Source/JavaScriptCore/inspector/remote/glib/RemoteInspectorGlib.cpp

```diff
@@ -94,7 +94,6 @@ void RemoteInspector::stopInternal(StopSource)
 
     updateHasActiveDebugSession();
 
-    m_automaticInspectionPaused = false;
```

Comment:
does this need to `m_pausedAutomaticInspectionCandidates.clear()`?

Reply:
There isn't currently a way for glib (or the socket impl below) to ever add stuff to this list, but for completeness it can't hurt.